src/test/java/domain/item/ItemsTest.java

```diff
@@ -0,0 +1,28 @@
+package domain.item;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThatNoException;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class ItemsTest {
+
+    @Test
+    @DisplayName("게임 참가자 수와 입력된 실행 결과의 수가 일치하지 않으면 예외를 발생한다.")
+    void createItems_success() {
+        List<String> items = List.of("꽝", "5000", "꽝", "3000");
+        assertThatNoException().isThrownBy(() -> new Items(4, items));
+    }
+
+    @Test
+    @DisplayName("게임 참가자 수와 입력된 실행 결과의 수가 일치하지 않으면 예외를 발생한다.")
+    void createItems_fail() {
+        List<String> items = List.of("꽝", "5000", "꽝", "3000", "꽝");
+        assertThatThrownBy(() -> new Items(4, items))
+                .isInstanceOf(IllegalArgumentException.class)
+                .hasMessageContaining("참가자 수와 실행 결과의 수가 일치해야 합니다.");
+    }
```

Comment:
지금도 너무 좋지만 이렇게 검증할 수도 있을 것 같아요 😃
```suggestion
    @Test
    @DisplayName("게임 참가자 수와 입력된 실행 결과의 수가 일치하지 않으면 예외를 발생한다.")
    void createItems_success() {
        List<String> items = List.of("꽝", "5000", "꽝", "3000");
        int playerCount = items.size();
        assertThatNoException().isThrownBy(() -> new Items(playerCount, items));
    }

    @Test
    @DisplayName("게임 참가자 수와 입력된 실행 결과의 수가 일치하지 않으면 예외를 발생한다.")
    void createItems_fail() 
        List<String> items = List.of("꽝", "5000", "꽝", "3000", "꽝");
        int playerCount = items.size() + 1;
        assertThatThrownBy(() -> new Items(playerCount, items))
                .isInstanceOf(IllegalArgumentException.class)
                .hasMessageContaining("참가자 수와 실행 결과의 수가 일치해야 합니다.");
    }
```

Reply:
넵 감사합니다! playerCount 라는 변수를 사용해서 Items를 초기화하니 코드만으로 테스트의 의도를 더 잘 파악할 수 있어서 더 좋은 방법인 것 같습니다!